columnphysics/icepack_therm_vertical.F90

```diff
@@ -107,7 +107,7 @@ subroutine thermo_vertical (dt,          aicen,     &
                                   congel,      snoice,    &
                                   mlt_onset,   frz_onset, &
                                   yday,        dsnow,     &
-                                  prescribed_ice)
+                                  prescribed_ice, alvl)
```

Comment:
Suggest moving alvl up in the calling arguments.  Makes no difference technically, but seems like it belongs with apnd and hpond, for instance.

Reply:
Done.